### DISCUSSION TITLE
readme was not updated

### DISCUSSION THREAD

**Discussion by zjc2684613** [verdaccio/verdaccio#2543] [Q&A] [answered]
I updated several versions of the plugin.
But when I switched versions, the file was not updated.
The file is still the latest version.
![image](https://user-images.githubusercontent.com/30221538/137887700-86a68a87-9ab8-4069-af34-a8c1a8f2356e.png)
![image](https://user-images.githubusercontent.com/30221538/137888169-b7391304-bc4d-4ab4-8292-56f82bef30d1.png)

**Comment by juanpicado** [comment]
@zjc2684613 please describe step by step what you did and 🥺 include a package.json, versions package managers etc. Otherwise,
hard to help. https://raw.githubusercontent.com/verdaccio/verdaccio/master/.github/ISSUE_TEMPLATE/bug_report.md

**Comment by mbtools** [comment, marked as answer]
Same as https://github.com/verdaccio/verdaccio/discussions/3899
